roles/iscsi/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Install iscsiadm
+  become: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - open-iscsi
+    - multipath-tools
+    - multipath-tools-boot
+- name: Configure iscsiadm startup
+  become: yes
+  lineinfile:
+    dest: /etc/iscsi/iscsid.conf
+    regexp: "^node.startup ="
+    line: "node.startup = automatic"
+- name: Exec connection
+  command: "{{ item }}"
+  become: yes
+  with_items:
+    - "iscsiadm -m discovery"
```

Comment:
I got a null return here.
You don't need to put IP as an option ("-p" option)?

Reply:
Ah, that part depends on implementation/configuration of iSCSI target side..
It seems, these operation would not be useful for installation, so let me remove this last task.